### DISCUSSION TITLE
Application keeps getting rejected

### DISCUSSION THREAD

**Discussion by e6d1fe** [community/community#119636] [GitHub Education]
### Hi everyone,

I'm trying to prove my academic status, but I keep getting rejected.

<img width="784" alt="image" src="https://github.com/community/community/assets/108079454/39c6cae9-4c45-46d6-8db4-06860581c072">

This is the message I get.
I've seen that locations are enabled on my browser multiple times, and I checked that the name of my institution is in the photo (I've tried both my student ID & certificate of attendance in English).
I have also canceled my application & restarted a new one each time I quit my browser and tried again.

It would be great if I could get any help regarding this problem. Thank you!

**Comment by eliz-amst** [comment]
I have the same problem, first double check the files you submit to them, that they have your name as in your billing address and your institution name, combine files together, resubmit the application, check your location settings to enable them for GitHub
